### ISSUE TITLE
Benchmarks fail to run on GHC 7.10.3

### ISSUE THREAD

**Issue by RyanGlScott** [takano-akio/fast-builder#3]
Steps to reproduce:

```
$ cabal get fast-builder
Unpacking to fast-builder-0.0.0.4/
$ cd fast-builder-0.0.0.4/
$ cabal bench
...
Running 3 benchmarks...
Benchmark aeson: RUNNING...
aeson: ../aeson/benchmarks/json-data/twitter100.json: openBinaryFile: does not exist (No such file or directory)
Benchmark aeson: ERROR
```

See also https://github.com/iu-parfunc/sc-haskell/issues/7

**Comment by takano-akio** [comment]
Yes, I should stop relying on `aeson` being in a sibling directory, obviously :)

I'll try to do something about this in 24 hours.

**Comment by takano-akio** [comment]
I believe this is fixed in 0.0.0.5. Thank you for reporting this!
